Building from current Dockerfile on Ubuntu 16.04 produces error.

When on Xubuntu 16.04 64-bit I run `docker build -t todolist .` I get at the end:

```
Step 8 : RUN go install github.com/gammons/todolist
 ---> Running in 8d7e755c5024
# github.com/gammons/todolist
/usr/lib/go/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/lib/gcc/x86_64-alpine-linux-musl/6.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: cannot find Scrt1.o: No such file or directory
/usr/lib/gcc/x86_64-alpine-linux-musl/6.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: cannot find crti.o: No such file or directory
/usr/lib/gcc/x86_64-alpine-linux-musl/6.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: cannot find -lpthread
/usr/lib/gcc/x86_64-alpine-linux-musl/6.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: cannot find -lssp_nonshared
collect2: error: ld returned 1 exit status
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,6 +1,4 @@
 FROM golang:alpine
-MAINTAINER Quey-Liang Kao <s101062801@m101.nthu.edu.tw>
-# Last Modified: 2017/01/21
 
 RUN apk update && apk add git
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,6 +9,8 @@ RUN mkdir -p /src/github.com/gammons/
 RUN go-wrapper download github.com/gammons/todolist
 RUN go-wrapper install github.com/gammons/todolist
 
+RUN ln -s /.todos.json $HOME/.todos.json
+
 CMD ["todolist", "web"]
 
 EXPOSE 7890

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,23 +1,14 @@
-FROM alpine
+FROM golang:alpine
 MAINTAINER Quey-Liang Kao <s101062801@m101.nthu.edu.tw>
 # Last Modified: 2017/01/21
 
-ENV PATH /bin:/sbin:/usr/bin
-
-RUN apk update \
-    && apk add go git
+RUN apk update && apk add git
 
 RUN mkdir -p /src/github.com/gammons/
 
-RUN git clone https://github.com/gammons/todolist.git \
-    && mv todolist /src/github.com/gammons/
-
-ENV GOPATH /
-
-RUN go install github.com/gammons/todolist
+RUN go-wrapper download github.com/gammons/todolist
+RUN go-wrapper install github.com/gammons/todolist
 
 CMD ["todolist", "web"]
 
 EXPOSE 7890
-
-

